docs/core/model_checkpoints.md

```diff
@@ -177,3 +177,56 @@ Please consider citing the following paper in any research manuscript using the
     journal={arXiv preprint arXiv:2311.00341},
 }
 ```
+
+
+## Open Materials 2024 (OMat24)
+
+* All config files for the OMat24 models are available in the [`configs/odac`](https://github.com/FAIR-Chem/fairchem/tree/main/configs/omat24) directory.
+* All models are equiformerV2 S2EFS models
+
+
+### OMat pretrained models
+
+These checkpoints are trained on OMat24 only. Note that predictions are *not* Materials Project compatible.
+
+| Model Name            | Checkpoint	| Config                                                                                      |
+|-----------------------|--------------|---------------------------------------------------------------------------------------------|
+| EquiformerV2-31M-OMat | [checkpoint](https://huggingface.co/fairchem/OMAT24/blob/main/eqV2_31M_omat.pt) | [config](https://github.com/FAIR-Chem/fairchem/tree/main/configs/omat24/all/eqV2_31M.yml)   |
```

Comment:
These links will hit a 401 to users who didn't first land on `https://huggingface.co/fairchem/OMAT24` and submit the access request form. 

Either let's add that detail at the start that people need to do that, or Let's remove the checkpoint paths here and just tell them to visit the HF page for model downloads.